### DISCUSSION TITLE
Parent Folder Button image

### DISCUSSION THREAD

**Discussion by GeorgeGeorgeGe** [rejetto/hfs#387] [General]
Hello,

I suggest changing the "parent folder" button image on the main page to be the up arrow (↑) instead of the left arrow (←) since the up arrow is the one used in all browser and in windows file explorer as well, and it is even used by HFS in another locations like in Admin >> Shared Files >> Add >> Add files or folders.
the left  arrow (←)  is often used for going one page back.
it is a little confusing for new users.
Please check.
Thanks.

**Comment by rejetto** [comment]
your considerations are correct and not new to me, but I considered even worse having 2 different function/buttons with "up arrow" in the same screen, the other one being "upload". So I opted for the left arrow instead.
